rclcpp/src/rclcpp/duration.cpp

```diff
@@ -222,6 +245,67 @@ Duration::operator*(double scale) const
       static_cast<long double>(rcl_duration_.nanoseconds) * scale_ld));
 }
 
+Duration &
+Duration::operator*=(double scale)
+{
+  *this = *this * scale;
+  return *this;
+}
+
+Duration
+Duration::operator*(const rclcpp::Duration & rhs) const
+{
+  bounds_check_mul(
+    this->rcl_duration_.nanoseconds,
+    rhs.rcl_duration_.nanoseconds);
+  return Duration::from_nanoseconds(
+    rcl_duration_.nanoseconds * rhs.rcl_duration_.nanoseconds);
+}
+
+Duration &
+Duration::operator*=(const rclcpp::Duration & rhs)
+{
+  *this = *this * rhs;
+  return *this;
+}
+
+Duration
+Duration::operator/(double scale) const
```

Comment:
user can do the same thing via `Duration::operator*(double scale) const`? scale can be like 0.5 (division by 2).